src/store/metadata/stub/content/topics.json

```diff
@@ -24,6 +24,28 @@
             "parent_topics": [
                 "https://staging.idpd.uk/topics/economy"
             ]
+        },
+        {
+            "@id": "https://staging.idpd.uk/topics/communication",
+            "@type": "dcat:theme",
+            "identifier": "communication",
+            "title": "Communication",
+            "description": "something describing communication",
+            "sub_topics": null,
+            "parent_topics": [
```

Comment:
these are very close but not quite right. At the moment its saying

communication is the **parent** of itandindustry && itandindustry is the **parent** of communicaiton

bit circular, needs to be either:

itandindustry is the **parent** of communication && communicaiton is the **child** of itandindustry
_or_
communication is the **parent** of itandindustry && itandindustry is the **child** of communication

think of it like flip sides of a coin, if (a) is defined as the parent of (b) then (b) must be defined as the child of (a).